server/database/Session.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * A object to indicate current session, currently, use for database session, like a transaction session.
+ * So with knowing the current database transaction session, we can easily break the current process, and
+ * rollback the operation, and if encounter any accidentally problem or failure, we can rollback all
+ * operations.
+ *
+ * Note: `dataMigration` is temporary property that lets Session object use a different database connection
+ * instead of the default database used for the service. This feature is introduced for domain migration
+ * project (monolith app/db into separate bounded domain services/dbs) that requires data to be written to
+ * select databases for backward compatability. These extra db write capabilities will be removed once
+ * services are succesfully separated from the monolith.
+ */
+
+const { knex, knexMainDB } = require('./knex');
+
+class Session {
+  constructor(dataMigration = false) {
```

Comment:
Since we don't need to have dataMigration logic in our session, this can be simplified to just have one `knex` object and remove references and the logic that sorrounds `knexMainDB`.